paynow/__init__.py

```diff
@@ -1,3 +1,3 @@
 from .model import * 
 
-__version__="0.0.1"
+__version__="1.0.5+1"

```

Comment:
Could we say this is a breaking change to bump the major ?

Reply:
not really, i just made a `PR` without changing the `personal` details i was experimenting on. But its definitely not